aldryn_forms/boilerplates/legacy/templates/aldryn_forms/base_form.html

```diff
@@ -0,0 +1,18 @@
+{# This is a very nasty hack to redirect the user #}
+{% if form_success_url %}
+<script type="text/javascript">
+    window.location.replace("{{ form_success_url }}");
```

Comment:
@jmelett indeed we should, sadly this is one of the limitations we have when working with plugins. They can't return a response (in this case a redirect).